StatEngine/stat-engine
# Exposure Page

## Overview
Add an exposure page under Applications that exposes NFORS mobile application information and a way for users to get their departments IAC.

## GitHub Issues
- #442 

## Changes
- New Controllers
- New Views
- New endpoint

## Screenshots / Videos
<img width="1452" alt="Screen Shot 2020-04-08 at 10 11 09 PM" src="https://user-images.githubusercontent.com/2092432/78851073-e0d95580-79e6-11ea-8662-275c31e59ff0.png">

<img width="1170" alt="Screen Shot 2020-04-08 at 10 11 14 PM" src="https://user-images.githubusercontent.com/2092432/78851079-e3d44600-79e6-11ea-8136-b11b137fbe0c.png">

## Note
- PR in NFORS repo must be merged first

## Steps to Test
- Ensure environment variables are NFORS environment variables are set
- Login
- Click Exposures under the Applications tab in side menu
